docs/configuration.md

```diff
@@ -165,12 +165,31 @@ If you have 16 cores, you could (loosely spoken) assign 14 of them to mining usi
 mining:
     beneficiary: "beneficiary_pubkey_to_be_replaced"
     cuckoo:
-        miner:
-            edge_bits: 29
-            executable: mean29-avx2
-            extra_args: -t 14
+        edge_bits: 29
+        miners:
+            - executable: mean29-avx2
+              extra_args: -t 14
 ```
 
+#### Combining different miners
+
+Your mining setup may also contain multiple miners, which will be run simultaneously by your node.
+For example, to combine CPU miner with CUDA miner the following configuration can be used:
+```yaml
+mining:
+    beneficiary: "beneficiary_pubkey_to_be_replaced"
+    cuckoo:
+        edge_bits: 29
+        miners:
+            - executable: mean29-generic
+              extra_args: -t 2
+            - executable: cuda29
```

Comment:
Without `executable_group: aecuckooprebuilt` this will not work unless user built miner, and it seems a misleading assumption in an example (that) that in a doc (this) separate from the main cuda-miner.md

More generally: Can this section be deleted altogether, or reduced to a single sentence redirecting the user to the cuda-miner.md (that - I double checked - is included in the release binary as per rebar.config)? I am concerned that too many example configurations are too costly to maintain as correct examples (unless we automate that, that is a out of scope).